CHANGELOG.md

```diff
@@ -2,10 +2,10 @@
 
 **Breaking changes**
 
-* [CHANGE]
+* [CHANGE] Rename `node_package_throttle` label `node` to `package`. #871
```

Comment:
This should be just listed under the breaking changes section, we typically list one item per PR.